app/features/syllabus/core.py

```diff
@@ -0,0 +1,30 @@
+from services.tool_registry import ToolFile
+from services.logger import setup_logger
+from features.syllabus.tools import SyllabusBuilder
+from api.error_utilities import LoaderError, ToolExecutorError
```

Comment:
Fetch these utility functions from the `app` folder.

Reply:
fixed it